modules/integration_aws-elasticache-redis/conf/05-network-con-allowance.yaml

```diff
@@ -0,0 +1,15 @@
+module: AWS ElastiCache redis
```

Comment:
Nit: I'd name the file `05-network-conntrack-allowance.yaml`

Reply:
Done :checkered_flag: